std/std.zig

```diff
@@ -30,6 +30,7 @@ pub const hash = @import("hash.zig");
 pub const hash_map = @import("hash_map.zig");
 pub const heap = @import("heap.zig");
 pub const io = @import("io.zig");
+pub const linked_list = @import("linked_list.zig");
```

Comment:
I liked it the other way, `std.LinkedList`.

Reply:
This allows people to write: `std.linked_list.TailQueue` vs `std.linked_list.SinglyLinkedList`, etc.
My thought was that `std.LinkedList` may get removed.